components/pkg-export-docker/src/main.rs

```diff
@@ -43,5 +43,5 @@ fn start(ui: &mut UI) -> Result<()> {
     let m = cli.get_matches();
     debug!("clap cli args: {:?}", m);
 
-    export_docker::export_for_cli_matches(ui, &m)
+    export_docker::export_for_cli_matches(ui, &m).map(|_| ())
```

Comment:
It seems like nothing in the `start` function raises an error, so does its return type make sense? Maybe it should just have none, and then you woudn't have to use this `map` - which looks kinda weird, you're basically saying: iterate over every result, and completely ignore them.

Reply:
`export_to_cli_matches` used to return `Result<()>`, now it returns `Result<DockerImage>`, so we need to turn it back to `Result<()>`. This can be done that way or with `.and(Ok(()))`, or with:
```
export_docker::export_for_cli_matches(ui, &m)?;
Ok(())
```

Reply:
@asymmetric what @krnowak said and 'map' on Result and Option doesn't iterate, just maps one content to another.

I like the `and(Ok())` better than the explicit separate return.

Reply:
After changing, I'm not longer sure it looks better:

-    export_docker::export_for_cli_matches(ui, &m).map(|_| ())
+    export_docker::export_for_cli_matches(ui, &m).and(Ok(()))

meh. :)

Reply:
I think the second one conveys the meaning better.

Reply:
hmm... maybe but the ugliness of `Ok(())` annoys me more. :)

Reply:
It's beautiful.